.circleci/config.yml

```diff
@@ -8,6 +8,38 @@ orbs:
   sonarcloud: sonarsource/sonarcloud@1.0.2
   slack: circleci/slack@3.4.2
 jobs:
+  tag-and-release:
+    docker:
+      - image: cimg/base:2020.01
+    environment:
+      TAG_FORMAT: "{Major}.{Minor}.{Patch}"
+      GITHUB_OWNER: csu-booking-platform
+      GITHUB_REPO: application
```

Comment:
Wouldn't the owner be the CSU and the repo be the CSU-booking-platform?

Reply:
Its the github organization name & repo name. 

The https://github.com/csu-booking-platform/application belongs to the org we created for capstone: https://github.com/csu-booking-platform